docs/release-notes/rockcraft-1-14.rst

```diff
@@ -0,0 +1,160 @@
+
+     name = "craft-cli"
+    -version = "3.0.0"
+    +version = "3.1.2"
+
+     name = "craft-grammar"
+    -version = "2.0.3"
+    +version = "2.2.0"
+
+     name = "craft-parts"
+    -version = "2.16.0"
+    +version = "2.20.1"
+
+    docs: add maven-use reference page by @bepri in #1179
+    feat(jlink): introduce add_modules parameter by @vpa1977 in #1169
+    fix(overlay): detect conflicts between overlay and install by @tigarmo in #1190
+
+Rockcraft 1.14 release notes
+============================
+
+29 August 2025
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.14.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+Requirements and compatibility
+------------------------------
+
+To run Rockcraft, a system requires the following minimum hardware and
+installed software. These requirements apply to local hosts as well as VMs and
+container hosts.
+
+
+Minimum hardware requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- AMD64, ARM64, ARMv7-M, RISC-V 64-bit, PowerPC 64-bit little-endian, or S390x
+  processor
+- 2GB RAM
+- 10GB available storage space
+- Internet access for remote software sources and the Snap Store
+
+
+Platform requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 3 3
+
+  * - Platform
+    - Version
+    - Software requirements
+  * - GNU/Linux
+    - Popular distributions that ship with systemd and are `compatible with
+      snapd <https://snapcraft.io/docs/installing-snapd>`_
+    - systemd
+
+
+What's new
+----------
+
+Rockcraft 1.14 brings the following features, integrations, and improvements.
+
+New ``entrypoint-command`` project key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This key defines the rock's OCI entrypoint, bypassing the Pebble-powered entrypoint
+defined by Rockcraft. This can be necessary for cases like OS and base images, where
+an application-centric Pebble service is inadequate.
+
+See the :ref:`documentation <rockcraft-yaml-entrypoint-command>` for details.
+
+Minor features
+--------------
+
+Rockcraft 1.14 brings the following minor changes.
+
+Improved ``test`` command
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``test`` command now packs and tests all platforms that can be built on the running
+environment.
+
+Improved keyserver support in ``package-repositories``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If an attempt to connect to the default keyserver to retrieve a public key fails, the
+package-repositories mechanism will now try to connect through the standard port 80
+while also respecting proxy addresses set via the standard ``http_proxy`` and
+``https_proxy`` environment variables.
+
+New ``jlink-extra-modules`` key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This new :ref:`craft_parts_jlink_plugin` key allows defining additional modules that
+should be included in the generated OpenJDK image.
```

Comment:
As another new key, should this be in new features? I get that this isn't as significant as the other, but that's a judgement that might seem arbitrary to users.

Reply:
sure, sounds good